src/Devito.jl

```diff
@@ -274,6 +274,15 @@ struct SubDomain{N}
     o::PyObject
 end
 
+"""
+    name(subdomain)
+
+returns the name of the subdomain
+"""
+function name(x::SubDomain{N}) where {N}
```

Comment:
```suggestion
function name(x::SubDomain)
```
There is no need to specify N here.